src/main/java/medicaldoctor/entities/Patient.java

```diff
@@ -0,0 +1,99 @@
+package medicaldoctor.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import medicaldoctor.core.DatabaseScope;
+import medicaldoctor.util.HashAndSalt;
+import org.hibernate.query.Query;
+
+@Entity
+@Table(name = "Patient", uniqueConstraints = {
+    @UniqueConstraint(columnNames = "Id")
+    , @UniqueConstraint(columnNames = "Full Name")})
```

Comment:
full name is not a field in this entity, nor should it be a unique if it did exist.